sources/packages/backend/apps/db-migrations/src/sql/InstitutionLocations/Create-institution-locations-history-table.sql

```diff
@@ -0,0 +1,64 @@
+-- Table to keep historical data.
+-- Columns history_timestamp and history_operation should be default columns for all the history tables.
+-- Columns other than the above-mentioned should reflect the columns from the original table without constraints and relationships.
+-- The order of the columns below follow the exact same order from the existing table.
+CREATE TABLE sims.institution_locations_history (
+  history_timestamp TIMESTAMP WITH TIME ZONE NOT NULL,
+  history_operation VARCHAR(20) NOT NULL,
+  id INT,
+  name VARCHAR(100),
+  info JSONB,
+  institution_id INT,
+  created_at TIMESTAMP WITH TIME ZONE,
+  updated_at TIMESTAMP WITH TIME ZONE,
+  creator INT,
+  modifier INT,
+  institution_code CHAR(4),
+  primary_contact JSONB,
+  has_integration BOOLEAN,
+  integration_contacts VARCHAR(300) []
+);
+
+CREATE INDEX institution_locations_history_timestamp ON sims.institution_locations_history(history_timestamp);
+
+COMMENT ON INDEX sims.institution_locations_history_timestamp IS 'Historical data index to improve point-in-time data retrieval.';
```

Comment:
Minor and not blocker at all. I missed in previous PR that index did not start with prefix idx.

Reply:
Good point, I searched by "create index" on the source code and 3 out of 15 places are using "idx" prefix. I do not believe that this would be the pattern. Thoughts?

Reply:
I see, you are right and it is not consistently followed. Resolving the comment.